src/commands/edge/drivers/package.ts

```diff
@@ -56,12 +56,25 @@ export default class PackageCommand extends EdgeCommand {
 	static flags = {
 		...EdgeCommand.flags,
 		...outputItem.flags,
-		'dry-run': flags.boolean({
-			char: 'd',
-			description: 'save package to edge.zip file instead of uploading it',
+		'build-only': flags.string({
```

Comment:
Technically this makes this a breaking change but the output of `--dry-run` was only useful for debugging so what we're breaking is just a debug option. (And it's still possible, the user just needs slightly different options.